jade/distribution.yaml

```diff
@@ -462,7 +462,7 @@ repositories:
       version: 1.0.2-1
     source:
       type: git
-      url: https://github.com/euslisp/jskeus.git
+      url: https://github.com/tork-a/jskeus-release.git
       version: master
```

Comment:
I don't know what you're doing here, but pointing to the master branch won't work.

Why are you patching in the `CMakeLists.txt` instead of putting it in the upstream repository?